Project.toml

```diff
@@ -18,9 +18,9 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 BinaryProvider = "^0.5.3"
 CMake = "^1.1.1"
-CxxWrap = "^0.9.0"
+CxxWrap = "0.9,0.10"
 JSON = "^0.20, ^0.21"
-julia = "^1.0"
+julia = "^1.3"
```

Comment:
Just out of curiosity: since `^` is the default anyway, is there a reason you specify it explicitly in most of these compat entries?